cocos/physics/framework/deprecated.ts

```diff
@@ -36,6 +36,21 @@ replaceProperty(PhysicsSystem.prototype, 'PhysicsSystem.prototype', [
 removeProperty(PhysicsSystem.prototype, 'PhysicsSystem.prototype', [
     {
         "name": "useFixedTime"
+    },
+    {
+        "name": "useCollisionMatrix"
+    },
+    {
+        "name": "updateCollisionMatrix"
+    },
+    {
+        "name": "resetCollisionMatrix"
+    },
+    {
+        "name": "isCollisionGroup"
+    },
+    {
+        "name": "setCollisionGroup"
```

Comment:
We can't remove these public APIs directly, please do a deprecation with the same functionality instead.